source/photo.html

```diff
@@ -0,0 +1,183 @@
+          <p class="userphotos-list__comment">
+            <span class="userphotos-list__comment-name">Сергей</span>
+            <span class="userphotos-list__comment-time">(25 часов назад)</span>
+            Самая кубанская в мире!
+          </p>
+          <button class="like-button">
+            <img src="https://via.placeholder.com/150x150" alt="лайк">
+          </button>
+          <span class="userphotos-list__comment-like">Нравится: 4</span>
+        </li>
+        <li class="userphotos-list__item">
+          <img src="https://via.placeholder.com/150x150" alt="фото">
+          <p class="userphotos-list__comment">
+            <span class="userphotos-list__comment-name">Максим</span>
+            <span class="userphotos-list__comment-time">(40 часов назад)</span>
+            Где снег-то?
+          </p>
+          <button class="like-button">
+            <img src="https://via.placeholder.com/150x150" alt="лайк">
+          </button>
+          <span class="userphotos-list__comment-like">Нравится: 150</span>
+        </li>
+        <li class="userphotos-list__item">
+          <img src="https://via.placeholder.com/150x150" alt="фото">
+          <p class="userphotos-list__comment">
+            <span class="userphotos-list__comment-name">Надежда</span>
+            <span class="userphotos-list__comment-time">(45 часов назад)</span>
+            Всем GM и взаимные лайки!
+          </p>
+          <button class="like-button">
+            <img src="https://via.placeholder.com/150x150" alt="лайк">
+          </button>
+          <span class="userphotos-list__comment-like">Нравится: 2560</span>
+        </li>
+      </ul>
+
+      <p class="photo-text">
+        Затмите их всех!
+        Выкладывайте все, что накопилось
+        в телефоне!
+      </p>
+
+      <div class="sharing">
+        <img src="https://via.placeholder.com/150x150" alt="фото">
+        <div class="sharing-tools">
+          <ul class="sharing-tools__list">
+            <li class="sharing-tools__item sharing-tools--item-cut"></li>
+            <li class="sharing-tools__item sharing-tools--item-fill"></li>
+            <li class="sharing-tools__item sharing-tools--item-bright"></li>
+          </ul>
+          <div class="sharing-range">
+            <div class="sharing-range__scale"></div>
+            <div class="sharing-range__toggle"></div>
+          </div>
+        </div>
+        <div class="crossline"></div>
+        <div class="button-wrap">
+          <button class="sharing-button" type="submit">Запостить</button>
+          <button class="sharing-button" type="reset">Отмена</button>
+        </div>
+      </div>
+    </main>
+
+    <footer class="page-footer">
+      <a class="footer-logo" href="#">
+        <img src="https://via.placeholder.com/150x150" alt="логотип">
+      </a>
+      <ul class="social-list">
+        <li class="social-list__link">
+          <a class="social-list__link--twiter" href="#"></a>
+        </li>
```

Comment:
```suggestion
        <li class="social-list__item">
          <a class="social-list__link social-list__link--twiter" href="#"></a>
        </li>
```
и далее, по аналогии.